src/services/refactors/convertExport.ts

```diff
@@ -23,6 +23,9 @@ namespace ts.refactor {
             if (!info) return emptyArray;
 
             if (!isRefactorErrorInfo(info)) {
+                if (!(isIdentifier(info.exportName) && context.program.getTypeChecker().getSymbolAtLocation(info.exportName))) {
+                    return emptyArray;

```

Comment:
Check with @jessetrinity on this; it might be preferable to edit `getInfo` and return a `notApplicableReason` for this. Also, why do we need to check that we can resolve a symbol here?

Reply:
Re the question: later in `changeImports`, if `getSymbolAtLocation()` returns undefined, it throws a debugging failure.  (This happens with the third test: `/*[| |]*/ /** x */ export default x;`)

I initially made it just ignore these cases, but then I thought that the assertion means that it really wants to get there only when it can do something so I moved the test to here.  If it makes more sense to do nothing when it sees later that it doesn't resolve, then I'll move it.

Reply:
Oh, I completely missed the parentheses wrapping both conditions—I read this as “if it’s not an identifier but there _is_ a symbol there,” hence my confusion. Returning early when there’s _not_ a resolvable symbol makes total sense. My bad.

Reply:
Yep, returning a `RefactorErrorInfo` from `getInfo` would be preferable, something like
`{ error: getLocaleSpecificMessage(Diagnostics.Can_only_convert_named_export) }`